devices.go

```diff
@@ -255,10 +256,8 @@ func addDevicesToClearBlade(devices []*gcpiotpb.Device, deviceConfigs map[string
 }
 
 func updateDevice(device *gcpiotpb.Device) error {
-	transformedDevice := map[string]interface{}{
-		"device": transform(device),
-	}
 
+	transformedDevice := transform(device)
```

Comment:
I'm not sure how this worked before but currently the devicesPatch API expects the transformedDevice to not be nested